app/hooks/useFlightStatus.tsx

```diff
@@ -0,0 +1,103 @@
+import { q } from "@thorium/context/AppContext";
+
+/**
+ * Custom hook for accessing flight status information.
+ * 
+ * @description
+ * Provides a convenient interface for accessing flight state including
+ * whether a flight is active, paused, and basic flight information.
+ * 
+ * @returns {FlightStatusReturn} Flight status data and helper flags
+ * 
+ * @example
+ * ```typescript
+ * const { isActive, isPaused, flightName } = useFlightStatus();
+ * 
+ * if (isPaused) {
+ *   console.log("Flight is paused");
+ * }
+ * ```
+ * 
+ * @see {@link q.flight.active} - Server procedure for flight data
+ * @since 1.0.0
+ */
+export function useFlightStatus() {
+  const [flight] = q.flight.active.useNetRequest();
```

Comment:
All of this information is pretty much already accessible from the `q.flight.active` request. This hook is just an extra hoop to jump through. Let's keep it simple and remove this hook please.